pkg/providers/aws/provider_redis.go

```diff
@@ -620,19 +620,27 @@ func buildRedisGenericMetricLabels(r *v1alpha1.Redis, clusterID, cacheName strin
 	labels["instanceID"] = cacheName
 	labels["productName"] = r.Labels["productName"]
 	labels["strategy"] = redisProviderName
-	labels["statusAWS"] = ""
-	labels["statusPhase"] = ""
 	return labels
 }
 
 // adds extra information to labels around resource
 func buildRedisInfoMetricLables(r *v1alpha1.Redis, group *elasticache.ReplicationGroup, clusterID, cacheName string) map[string]string {
 	labels := buildRedisGenericMetricLabels(r, clusterID, cacheName)
-	if group != nil && len(string(r.Status.Phase)) != 0 {
-		labels["statusAWS"] = *group.Status
+	if group != nil {
+		labels["status"] = *group.Status
+		return labels
+	}
+	labels["status"] = fmt.Sprintf("%s is nil", cacheName)
```

Comment:
I noticed that some of these are simply `"nil"` and others are a string like `"x is nil"`, would probably be nicer to keep just one for consistency

Reply:
agree, as the other labels already have identifiers like cacheName, will remove `x is nil` as metrics are easier to search for in Prometheus when they are consistent.